### DISCUSSION TITLE
Row Index / Row Number when selected

### DISCUSSION THREAD

**Discussion by NiranjanShah** [bosskmk/pluto_grid#188] [Q&A]
in one of the discussion u had replied about the currently selected row
"If you run mode as select, you can handle the event when the currently selected row is tapped or the enter key is pressed with the onSelected callback."

The onSelectedCallBack is happening only on **Enter KEY** & **Mouse Double Click** in Desktop/Web App.

However what happens when the user selects the row by **Mouse Single Click** or **Changes the row thru up/down arrow**.

is there any event like "RowFocusChanged" which fires on change of row.

PS: this is for Web/Desktop application.

**Comment by bosskmk** [comment]
Features should be added.
